src/battle_anim.c

```diff
@@ -435,30 +435,33 @@ static void Cmd_unloadspritegfx(void)
 
 static u8 GetBattleAnimMoveTargets(u8 battlerArgIndex, u8 *targets)
 {
-    u8 numTargets = 1;
+    u8 numTargets = 0;
+    int idx = 0;
+    u32 battler = gBattleAnimArgs[battlerArgIndex];
     switch (GetBattlerMoveTargetType(gBattleAnimAttacker, gAnimMoveIndex))
     {
     case MOVE_TARGET_BOTH:
-        targets[0] = gBattleAnimArgs[battlerArgIndex];
-        numTargets = 1;
-        if (IsBattlerAlive(BATTLE_PARTNER(targets[0])))
-        {
-            targets[1] = BATTLE_PARTNER(targets[0]);
-            numTargets = 2;
+        if (IsBattlerAlive(battler)) {
```

Comment:
Since it's the same for both cases I think this go above the switch statement as to not duplicate the code.